source/DG.HostApp/Components/ClusterConfigEditor/ApplicationTypeSourcesCard.razor

```diff
@@ -0,0 +1,82 @@
+﻿@using DG.Core.Model.ClusterConfig
+@using DG.HostApp.Components.ClusterConfigEditor.Forms
+
+<div class="card-body">
+    <table class="table">
+        <tr>
+            <th> Name: </th>
+            <th> Path: </th>
+            <th> Path type: </th>
+            <th> </th>
+            <th> </th>
+        </tr>
+        @foreach (var applicationTypeSource in ClusterConfig.ClusterDefinition.ApplicationTypesSources)
+
+        {
+    <tr>
+        <td>@applicationTypeSource.Name</td>
+        <td>@applicationTypeSource.Path </td>
+        <td>@applicationTypeSource.PathType"</td>
+        <td> <button class=" btn btn-outline-primary" @onclick="() => SetTypeSource(applicationTypeSource) ">Edit</button></td>
```

Comment:
Could we make Edit button to toggle edit menu?

Reply:
Edit button expands edit menu, when we click on it, and collapses it when we click submit button (we can rename it to "Ok" as you suggested).